### DISCUSSION TITLE
Pylontech CanBus

### DISCUSSION THREAD

**Discussion by iukbox** [hoylabs/OpenDTU-OnBattery#276] [Q&A]
Welche Hardware ist denn für den Pylontech CanBus vorgesehen, bzw. mit welcher Hardware wurde entwickelt / getestet.
Ich möchte gerne den Adapter mit auf eine Platine setzen.

**Comment by helgeerbe** [comment]
Schau mal hier #128

**Comment by madmartin** [comment]
Ich arbeite erfolgreich mit #226. Funktioniert einwandfrei.
